Poker.BE/Poker.BE.Domain.Tests/Game/RoomTests.cs

```diff
@@ -30,5 +32,78 @@ public void RoomTest()
             TestContext.WriteLine("end of roomTest");
         }
 
+        [TestMethod()]
+        //[ExpectedException(typeof(NotEnoughPlayersException))]
```

Comment:
[mstest expected exception](http://www.bradoncode.com/blog/2012/01/asserting-exceptions-in-mstest-with.html)
for example:
![image](https://cloud.githubusercontent.com/assets/9889268/26124233/685c0780-3a86-11e7-946c-9ac6119f775b.png)